ui/app/AppLayouts/Profile/Sections/Ens/Welcome.qml

```diff
@@ -0,0 +1,275 @@
+                text: qsTr("ENS names transform those crazy-long addresses into unique usernames.")
+                anchors.top: title.bottom
+                anchors.topMargin: 24
+                font.pixelSize: 14
+                anchors.left: parent.left
+                anchors.right: parent.right
+                horizontalAlignment: Text.AlignHCenter
+                wrapMode: Text.WordWrap
+            }
+
+            StyledText {
+                id: element1Number
+                text: "1"
+                anchors.left: parent.left
+                anchors.leftMargin: 24
+                anchors.top: subtitle.bottom
+                anchors.topMargin: 24
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element1Title
+                text: qsTr("Customize your chat name")
+                anchors.left: element1Number.right
+                anchors.leftMargin: 24
+                anchors.top: subtitle.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element1Subtitle
+                text: qsTr("An ENS name can replace your random 3-word name in chat. Be @yourname instead of %1.").arg(profileModel.profile.username)
+                anchors.left: element1Number.right
+                anchors.leftMargin: 24
+                anchors.top: element1Title.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element2Number
+                text: "2"
+                anchors.left: parent.left
+                anchors.leftMargin: 24
+                anchors.top: element1Subtitle.bottom
+                anchors.topMargin: 24
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element2Title
+                text: qsTr("Simplify your ETH address")
+                anchors.left: element2Number.right
+                anchors.leftMargin: 24
+                anchors.top: element1Subtitle.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element2Subtitle
+                text: qsTr("You can receive funds to your easy-to-share ENS name rather than your hexadecimal hash (0x...).")
+                anchors.left: element2Number.right
+                anchors.leftMargin: 24
+                anchors.top: element2Title.bottom
+                anchors.topMargin: 24
```

Comment:
You use `24` a lot. It would be a good case for a prop at the top of the component

Reply:
Yeah, but dunno if it's worth to do the change, because there is no design for the ens section yet. I just took the content from status-react and did not pay too much attention to the layout